src/options.h

```diff
@@ -113,9 +113,9 @@
 #  define MEMDEBUG
 #  define KEEP_BODIES
 #  endif
-// Memory sanitizer also needs thread-local storage
+// Memory sanitizer also needs small memory model
```

Comment:
No, it needs thread local storage. Please don't keep changing this. Being able to use memory sanitizer is important.